src/test/java/com/pnu/dev/shpaltaif/integration/service/PublicAccountServiceIntegrationTest.java

```diff
@@ -8,183 +8,164 @@
 import com.pnu.dev.shpaltaif.integration.BaseIntegrationTest;
 import com.pnu.dev.shpaltaif.repository.UserRepository;
 import com.pnu.dev.shpaltaif.service.PublicAccountService;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Autowired;
-
-import java.util.List;
+import org.springframework.test.annotation.Rollback;
 
 import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 
-public class PublicAccountServiceIntegrationTest extends BaseIntegrationTest {
-
-    private static final String NAME = "name";
-
-    private static final String SURNAME = "surname";
-
-    private static final String PROFILE_IMAGE_URL = "profileImageUrl";
-
-    private static final String DESCRIPTION = "description";
 
-    private static final String UPDATED_NAME = "updatedName";
-
-    private static final String UPDATED_SURNAME = "updatedSurname";
-
-    private static final String UPDATED_PROFILE_IMAGE_URL = "updatedProfileImageUrl";
-
-    private static final String UPDATED_DESCRIPTION = "updatedDescription";
+public class PublicAccountServiceIntegrationTest extends BaseIntegrationTest {
 
     @Autowired
     private PublicAccountService publicAccountService;
 
     @Autowired
     private UserRepository userRepository;
 
-    @Test
-    public void findByIdNotFound() {
+    private User userWriter;
 
-        assertEquals(0, publicAccountService.findAll().size());
+    @BeforeEach
```

Comment:
This hasn't be `@BeforeEach`. Make it a first test, otherwise we lose any performance improvement.